app/models/user.rb

```diff
@@ -71,4 +75,38 @@ def user_setting
   def confirmation_required?
     false
   end
+
+  private
+
+  def connect_to_mailchimp
```

Comment:
Assignment Branch Condition size for connect_to_mailchimp is too high. [23.54/15]